src/SectionsNavigation.Uno.WinUI/SectionsNavigation.Uno.WinUI.csproj

```diff
@@ -25,8 +25,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Uno.WinUI" Version="4.4.13" />
```

Comment:
I think this might be a breaking change because it forces the users to update to that specific version of Uno.

Reply:
I added this to the list of breaking changes and marked the commit as breaking with the ! signifier